project/interface.go

```diff
@@ -37,6 +37,7 @@ type APIProject interface {
 	CreateService(name string) (Service, error)
 	AddConfig(name string, config *config.ServiceConfig) error
 	Load(bytes []byte) error
+	ListStoppedContainers(ctx context.Context, services ...string) ([]string, error)
```

Comment:
Are you okay with merging this as-is and then iterating on container listing functionality later? It seems separate from this, so maybe we can address it in a different PR.